src/hash/calcHash.js

```diff
@@ -1,3 +1,14 @@
+import * as fs from 'fs/promises';
+import { createHash } from 'node:crypto';
+
 export const calculateHash = async () => {
-    // Write your code here 
-};
\ No newline at end of file
+  const hash = createHash('sha256');
+
+  fs.readFile('src/hash/files/fileToCalculateHashFor.txt')
+    .then(data => {
+      hash.update(data);
+      console.log(hash.digest('hex'));
+    });
+};
+
```

Comment:
calcHash.js - implement function that calculates SHA256 hash for file fileToCalculateHashFor.txt and return it as hex
function should return value, not print to console